### ISSUE TITLE
Creating an admin

### ISSUE THREAD

**Issue by michaelsomerville** [Repox/SimpleUsers#2]
Hey, Great starter script here! I am wondering how to identify a user as "Admin" ?

**Comment by michaelsomerville** [comment]
In addition - how do we configure a forgot password with this script ?

I am also, attempting to create a "sign as user" - have you had luck with this ?

**Comment by Repox** [comment]
Hi.

> Hey, Great starter script here! I am wondering how to identify a user as "Admin" ?

There is no "admin" user, but you could grant a user an attribute identifying them as an admin:

```
$simpleusers->setInfo('admin', 1);
```

And to test it:

```
if($simpleusers->getInfo('admin'))
{
    // is admin
}
```

>  In addition - how do we configure a forgot password with this script ?

You would have to manually create a process to this and then use the method `setPassword`. The script doesn't help you with all the mails and tokens necessary to do this in a secure way.

> I am also, attempting to create a "sign as user" - have you had luck with this ?

Well, it wouldn't be hard to make, by adding a new method to the script:

```
    public function loginAs($userId)
    {
        if($user = $this->getSingleUser($userId))
        {
            $_SESSION[$this->sessionName]["userId"] = $user["userId"];
            $this->logged_in = true;

            return true;
        }

        return false;
    }
```

**Comment by michaelsomerville** [comment]
Awesome! Thanks for the support and comments, much appreciated on that.

**Comment by michaelsomerville** [comment]
Quickly, would that sign as user ... dump a url ? Sorry, I am not an advanced coder. Lol

**Comment by Repox** [comment]
No, you would have to create a route to a URL yourself, where you use that specific method.

**Comment by michaelsomerville** [comment]
Okay, I am extremely lost with that function - where would I push that into the code ? and how would I call it.

**Comment by Tonenz16** [comment]
hi, i can't insert the function "only when you are admin must show a div in the index.php.
I used your method, but it doesn't work.
Could you give me some tips? Thank you!
